sdk/servicebus/service-bus/review/service-bus.api.md

```diff
@@ -157,20 +158,23 @@ export class ServiceBusClient {
     getDeadLetterReceiver(queueName: string, receiveMode: "receiveAndDelete"): Receiver<ReceivedMessage>;
     getDeadLetterReceiver(topicName: string, subscriptionName: string, receiveMode: "peekLock"): Receiver<ReceivedMessageWithLock>;
     getDeadLetterReceiver(topicName: string, subscriptionName: string, receiveMode: "receiveAndDelete"): Receiver<ReceivedMessage>;
-    getReceiver(queueName: string, receiveMode: "peekLock"): Receiver<ReceivedMessageWithLock>;
-    getReceiver(queueName: string, receiveMode: "receiveAndDelete"): Receiver<ReceivedMessage>;
-    getReceiver(topicName: string, subscriptionName: string, receiveMode: "peekLock"): Receiver<ReceivedMessageWithLock>;
-    getReceiver(topicName: string, subscriptionName: string, receiveMode: "receiveAndDelete"): Receiver<ReceivedMessage>;
-    getSender(queueOrTopicName: string): Sender;
+    getReceiver(queueName: string, receiveMode: "peekLock", options?: GetReceiverOptions): Receiver<ReceivedMessageWithLock>;
+    getReceiver(queueName: string, receiveMode: "receiveAndDelete", options?: GetReceiverOptions): Receiver<ReceivedMessage>;
+    getReceiver(topicName: string, subscriptionName: string, receiveMode: "peekLock", options?: GetReceiverOptions): Receiver<ReceivedMessageWithLock>;
+    getReceiver(topicName: string, subscriptionName: string, receiveMode: "receiveAndDelete", options?: GetReceiverOptions): Receiver<ReceivedMessage>;
+    // Warning: (ae-forgotten-export) The symbol "GetSenderOptions" needs to be exported by the entry point index.d.ts
+    getSender(queueOrTopicName: string, options?: GetSenderOptions): Sender;
     getSessionReceiver(queueName: string, receiveMode: "peekLock", options?: GetSessionReceiverOptions): SessionReceiver<ReceivedMessageWithLock>;
     getSessionReceiver(queueName: string, receiveMode: "receiveAndDelete", options?: GetSessionReceiverOptions): SessionReceiver<ReceivedMessage>;
     getSessionReceiver(topicName: string, subscriptionName: string, receiveMode: "peekLock", options?: GetSessionReceiverOptions): SessionReceiver<ReceivedMessageWithLock>;
     getSessionReceiver(topicName: string, subscriptionName: string, receiveMode: "receiveAndDelete", options?: GetSessionReceiverOptions): SessionReceiver<ReceivedMessage>;
     getSubscriptionRuleManager(topic: string, subscription: string): SubscriptionRuleManager;
 }
 
+// Warning: (ae-forgotten-export) The symbol "OptionsBag" needs to be exported by the entry point index.d.ts
+//
 // @public
-export interface ServiceBusClientOptions {
+export interface ServiceBusClientOptions extends OptionsBag {
     dataTransformer?: DataTransformer;
```

Comment:
@ramya-rao-a  - do we want to hide dataTransformer like we did in EventHubs?

Reply:
Yes, lets hide it for Preview 1. We will bring it back later based on how the serializer discussions go

Reply:
https://github.com/Azure/azure-sdk-for-js/pull/8106